edge/README_zh.md

```diff
@@ -0,0 +1,24 @@
+<!--
+ * @Author: zj.wang
+ * @Date: 2021-12-22 10:59:00
+ * @LastEditors: zj.wang
+ * @LastEditTime: 2021-12-22 11:56:26
+ * @Description: 
+-->
```

Comment:
remove this block content.

Reply:
ok

Reply:
LGTM, and could you please use `git rebase` to squash the commits to only one? that's more clearer.

Reply:
ok

Reply:
LGTM /assign @fisherxu